StringCalculatorTest/StringCalculatorKataTest.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using StringCalculator_Kata;
+namespace StringCalculatorTest
+{
+    [TestClass]
+    public class StringCalculatorKataTest
+    {
+        private StringCalculator _sc;
+        public StringCalculatorKataTest()
+        {
+            _sc = new StringCalculator();
+        }
+
+        [TestMethod]
+        [Description("Sum of numbers in empty string")]
+        [Owner("Lena")]
+        public void EmptyString()
+        {
+            //Arrange
+            int expectedNumber;
+            int actualNumber = 0;
+            //Act
+            expectedNumber = _sc.Add("");
```

Comment:
I think you should swap between expectedNumber and actualNumber 
because expecedNumber is your fact and you actualNumber should be the number you get from the calculation.